readme.md

```diff
@@ -230,6 +290,17 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
+var andy = Instructor.findOne({name: "Andy"}, function(err, doc){
+  if (!err) {
+    console.log("Success in retrieving " + doc.name);
+  } else {
+    console.log(err.message);
+  }
+});
+
+andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold"});
+andy.save(function (err) {
+  if (!err) console.log('Success!');
+});
 
+```
```

Comment:
might be better to nest the wishlist item creation inside of the callback for finding andy. IE:

```js
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```